n2y/main.py

```diff
@@ -95,11 +97,19 @@ def export_markdown(client, raw_rows, options):
             if pandoc_output:
                 markdown = pandoc.write(pandoc_output, format='gfm+tex_math_dollars') \
                     .replace('\r\n', '\n')  # Deal with Windows line endings
-                # reformat equations to have one backslash instead of 2
+
+                # normalize target path and create it if it doesn't exist
+                target = options.target
+                if not target[(len(target)-1)] == '/':
+                    target += '/'
```

Comment:
Do you need to add the `/` to the end of the path? I don't think `os.path.makedirs` should care if there is a path or not.

Reply:
it was moreso for the format string (`f"{options.target}{filename}.md"`) on line 112 but now that it uses `os.path.join` it's unnecessary.